core/src/main/java/io/camunda/migrator/impl/util/C7Utils.java

```diff
@@ -27,7 +27,7 @@ public static Map<String, FlowNode> getActiveActivityIdsById(ActivityInstance ac
     Arrays.asList(activityInstance.getChildActivityInstances()).forEach(actInst -> {
       activeActivities.putAll(getActiveActivityIdsById(actInst, activeActivities));
 
-      if (!SUB_PROCESS_ACTIVITY_TYPE.equals(actInst.getActivityType()) && !actInst.getActivityId().endsWith(MULTI_INSTANCE_BODY_SUFFIX)) {
```

Comment:
This is part of `getActiveActivityIdsById`.
`getActiveActivityIdsById` is used in 2 none test functions: 
- `activateMigratorJobs` -> this is called after validation, the PI has to be skipped already
- `validateProcessInstanceState` -> this is called just before validation, we want to retrieve multi instance activities, otherwise we could not validate them

Reply:
Do we also want to move the sub process check in line 30 into the validator? If so, we could open a follow up for that

Reply:
A subprocess is supported by migration there is some difference in how we activate it. This logic should stay here.